index.js

```diff
@@ -259,12 +259,20 @@ instance.prototype.handleTLS = function(data) {
 	self.CHOICES_TEMPLATES.length = 0;
 
 	for (var i = 0; i < data.length; ++i) {
-		var match = data[i].match(/^"([^"]+)"/);
-		if (match && match.length > 1) {
-			var file = match[1].replace(/\\/g, '\\\\');
-			self.CHOICES_TEMPLATES.push({ label: file, id: file });
-		} else {
-			var file = data[i].split(/ /)[0].replace(/\\/g, '\\\\');
+		// Template response parsing from SuperFlyTv/casparcg-connection
+		// https://github.com/SuperFlyTV/casparcg-connection/blob/master/src/lib/ResponseParsers.ts#L320
+		var match = data[i].match(/\"([\s\S]*)\" +([\s\S]*)/);
```

Comment:
Instead of [\s\S] in the first match, use [^"], and .* in the last

Reply:
That isn't the same. `[^"]*` does not match the string `"asdf"asdf"`, but the original regex does. But it could be optimized to `.*?` 
I intentionally copied this part from the other library to rely on some tested and proven code. These guys also have way more expirience with CasparCG than I do. But if you wan't me to change/optimize this code section I can do so.

Reply:
Yes, then you should rather do .* or .*? depending on what you want to do with the last match group there. Not sure what the last match group is for?

Reply:
The format of the TLS response in 2.0 and 2.1 always has this trailing part. It is there to safely detect the server version / used format.

Reply:
I'd write it as follows then, as you are not using the last group for anything.  `data[i].match(/\"(.*)\" .+/);`